plugins/discourse-calendar/app/serializers/discourse_post_event/basic_event_serializer.rb

```diff
@@ -38,42 +38,54 @@ def include_rrule?
     def rrule
       return nil unless include_rrule?
 
-      # Use the original event time (same as the frontend display) for rrule generation
+      rrule_timezone = object.show_local_time ? "UTC" : object.timezone
```

Comment:
Maybe there could be an `rrule_timzeone` method on the event object? Since you do this in here and in `event.rb`